NuKeeper/Configuration/CommandLineParser.cs

```diff
@@ -35,8 +35,18 @@ private static RepositoryModeSettings ReadSettingsForRepositoryMode(CommandLineA
             var githubToken = settings.GithubToken;
             var githubRepoUri = settings.GithubRepositoryUri;
 
+            Uri githubApiBase;
+            if (settings.GithubApiEndpoint != null)
+            {
+                githubApiBase = settings.GithubApiEndpoint;
```

Comment:
Isn't this what the default argument values are for? Have `github_api_endpoint=https://api.github.com` as default and you can override it if you need to?